### PR TITLE
Ghosts get plane masters

### LINKED ISSUE #23677: Ghosts do not get plane masters.
Showstopper for #23087 because I can't make lights render correctly if I can't rely on the plane master always existing.

### PR DESCRIPTION
**Pull request by ChangelingRain**
Fixes #23677
Disclaimer I don't know if this breaks anything so I _really_ hope it doesn't

### CODE REVIEW

**Comment by coiax** [comment]
what is a plane master

**Comment by ChangelingRain** [comment]
i don't know

File: code/_onclick/hud/ghost.dm [hunk 1]
```diff
@@ -77,6 +77,9 @@
 /datum/hud/ghost/show_hud()
 	var/mob/dead/observer/G = mymob
 	mymob.client.screen = list()
+	if(plane_masters.len)
```
Comment (RemieRichards) [inline code/_onclick/hud/ghost.dm]: not necessary, the for loop handles this internally.

**Comment by RemieRichards** [comment]
@coiax think of them as draw surfaces.

File: code/_onclick/hud/ghost.dm (hunk 1 above)
Comment (ChangelingRain) [inline code/_onclick/hud/ghost.dm]: i copied it from the hud code so i'll remove it there too

**Comment by MrPerson** [comment]
I independently fixed this in #21638. This is fine though, I'll have to resolve conflicts for unreverting anyway. I'm just pointing out that doing this was a good idea!

**Pull request merged**